src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.tsx

```diff
@@ -48,6 +49,7 @@ export type Props = {
 const ListedBlastSubmission = (props: Props) => {
   const { submission } = props;
 
+  const [isExpanded, setExpanded] = useState(false);
```

Comment:
We are using local state here, isn't it supposed to save the state? As of now I don't see it saving the expanded state.

Reply:
Do you know if we need to preserve the state?